tobler/tests/test_utils.py

```diff
@@ -43,9 +48,9 @@ def test_h3fy_clip():
         sac_hex.to_crs(32710).unary_union.area, 13131736346.537416, decimal=4
     )
 
+
+@pytest.mark.skipif(platform.system() == "Windows", reason='untested on windows')
```

Comment:
Can we specify the actual reason why this is skipped?

Reply:
> Can we specify the actual reason why this is skipped?

Great idea. @knaaptime can you add a simple note like:  `# See GH#174`

Reply:
I meant instead of
```py
reason='untested on windows'
```
we should have something like

```py
reason='windows suck and breaks floating point precision'
```
or whatever the reason for this test to fail is.

Reply:
Ahhh. LOL.

Reply:
the trouble is we're still not sure wtf is going on. The difference seems way too big to be precision? I think referencing this conversation is the smart thing to do

Reply:
But we can at least all agree that Windows sucks, right?